l1-l2-deposit-withdrawal/scripts/example.js

```diff
@@ -91,6 +96,17 @@ async function main() {
   const tx1 = await L1_ERC20.approve(L1StandardBridge.address, 1234)
   await tx1.wait()
 
+  // Sanity check, is the L2 address really the correct IL2StandardERC20 for this
+  // L1 token. If the contract doesn't implement IL2StandardERC20 we'll have an exception
+  // and the transfer will also fail.
+  //
+  // DO NOT remove this test. Without it a transfer might get blocked, and you won't get the
+  // assets back until seven days have passed. It might even be possible assets will be lost.
```

Comment:
This wording doesn't sound good. Maybe reword to:
  
"DO NOT remove this check.
Ensures L2 token compliance and validity. If the L2 token contract doesn't implement `IL2StandardERC20` or it does not correspond to the L1 token being deposited, an exception will occur and no deposit will take place. 
Alternatively the exception will occur once he deposit is relayed to L2 and the seven day wait period will apply for the bad deposit to be withdraw-able back on L1"